### ISSUE TITLE
Questions on Strategy, Governance & Operating Model.

### ISSUE THREAD

**Issue by jmanukonda** [MicrosoftDocs/architecture-center#1923]
Hello,

Do you have any documentation, or reference material or help for the questions below, as we plan on our Architecture design for chat bots?

1). The above picture shows the Azure Active directory validation at message level. How & where does the control exist for having the users authorize/ authenticate one time during the session active, and where do we control the duration of a session length

2). What are the recommendations from Microsoft on Authentication when a chat bot is being accessed by Internal customers (employees etc.) Vs external customers (Suppliers etc.). How does AD Vs Azure Active directory work in parallel?

3). Parameters to consider while choosing a channel (other than  the end users ability to access the channel), are there any functional elements to be considered while choosing a channel A Vs B?

4). What components of the architecture would allow us to control & govern the security aspects? - What is the recommendation from Microsoft to consider, and make sure we protect the data from PII/PCI/GDPR side.

Consider an example where I have a chat bot helping the Employees, Suppliers & Customers with FAQ's, as a part of which it collect the contact information. How do I secure the entity of Employee from Supplier/ Customers and vice-versa......can we secure entities by role?

5). COuld not download the content from git for the above, can you share this?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d0a5d586-36db-1258-a949-04084fa331e6
* Version Independent ID: 5615a84e-21d7-6a9c-6c2a-37bba2b9d5d8
* Content: [Information Chatbot - Azure Solution Ideas](https://docs.microsoft.com/en-us/azure/architecture/solution-ideas/articles/information-chatbot#feedback)
* Content Source: [docs/solution-ideas/articles/information-chatbot.md](https://github.com/microsoftdocs/architecture-center/blob/master/docs/solution-ideas/articles/information-chatbot.md)
* Service: **architecture-center**
* GitHub Login: @adamboeglin
* Microsoft Alias: **pnp**

**Comment by KrishnaG-MSFT** [comment]
@jmanukonda Thank you for your feedback! We will review and provide an update as appropriate.

**Comment by ketan-chawda-msft** [comment]
@MikeWasson any thoughts here?

**Comment by adamboeglin** [comment]
We're deprecating this article in favor of the much more detailed reference architecture here: https://docs.microsoft.com/en-us/azure/architecture/reference-architectures/ai/conversational-bot

Please review this article and open a new issue if you still have questions.
